Kanban/fix_items.md

```
@@ -0,0 +1,75 @@
+# Fix Kanban — Gate Blockers
+
+## To Do
+
+      - [x] Optional loop helpers: extract HITL handling/resume state into `loop_hitl_orchestrator.py` and iteration/output mappers into `loop_mapper.py`.  
+        Run: `pytest tests/application/core/test_executor_core_execute_loop.py tests/unit/test_loop_step_policy.py`
+
+## Done
+
+### Robustness fixes
+  - due: 2025-12-05
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)